keyboards/zwerg/keyboard.json

```
@@ -0,0 +1,98 @@
+{
+    "manufacturer": "floookay",
+    "keyboard_name": "zwerg",
+    "maintainer": "floookay",
+    "bootloader_instructions": "Press the reset button on the Gemini controller or right-side-bottom-right + esc",
+    "tags": ["ortho", "split", "40%", "OSHW"],
+    "development_board": "helios",
```

As can be seen in the repo linked in the readme, are not using a helios so should not use the preset
```suggestion
    "processor": "RP2040",
    "bootloader": "rp2040",
```